Documentation/applications/interpreters/wamr/index.rst

```diff
@@ -1,3 +1,23 @@
 ==================================
 ``wamr`` Webassembly Micro Runtime
 ==================================
+
+This captures how to try WAMR on NuttX.
+
+Preparing test.wasm
+===================
+
+Before trying WAMR, we need a test.wasm program, we need follow the "wasi-sdk" part in this `WAMR guide <https://wamr.gitbook.io/document/basics/getting-started/host_prerequsites>`_ to get the SDK and build the test.wasm program on our host.
```

Comment:
Maybe you could include a simple test.wasm example here

Reply:
The path to WASM sample is added in the updated version.